shared/common-adapters/input.desktop.tsx

```diff
@@ -273,7 +273,7 @@ class Input extends React.PureComponent<Props, State> {
     const defaultRowsToShow = Math.min(2, this.props.rowsMax || 2)
     const containerStyle = this._containerStyle(underlineColor)
 
-    const commonInputStyle = {
+    const commonInputStyle: any = {
```

Comment:
why are these `any` instead of the CSS type?

Reply:
some of these types were complicated by issues that weren't fixing at this phase of this project so i punted